docs/reference/tools-reference/open_source_llm_tool.md

```diff
@@ -2,48 +2,65 @@
 
 ## Introduction
```

Comment:
Please add doc to docs/reference/index.md

Reply:
I see it here: https://github.com/microsoft/promptflow/blob/main/docs/reference/index.md line 40.